src/styles/block/clock.scss

```diff
@@ -0,0 +1,54 @@
+@mixin handSize {
+  position: absolute;
+  bottom: 50%;
+  left: 40%;
+  border-radius: 10px;
+  width: 5px;
+}
+
+.clock {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+
+  &__center {
+    background-color: black;
+    height: 20px;
+    width: 20px;
+    border-radius: 50%;
+  }
+
+  @keyframes spin {
+    100% {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
also, you can create an animation.scss file to this code

Reply:
Agree, it would be better to create an `animation.scss` file